server/routes/handlers/prisonerSearchHandler2.test.ts

```diff
@@ -0,0 +1,72 @@
+import { getMockReq, getMockRes } from '@jest-mock/express'
+import nock from 'nock'
+import config from '../../config'
+import prisonerSearchHandler from './prisonerSearchHandler'
+
+const userToken = { access_token: 'token-1', expires_in: 300 }
+const nomsNumber = 'AA123AA'
+
+describe('prisonerSearchHandler', () => {
```

Comment:
Briefly, what are we doing with this test versus the previous - not `2` version?

Reply:
I wanted to show the two versions to everybody and discuss options.  Testing javascript/typsecript is a pain in my opinion, for people that haven't done much it might be useful.  And someone else may know better ways